src/components/header/Header.jsx

```diff
@@ -0,0 +1,16 @@
+import "./Header.css";
+import logo from "../../assets/logo.svg";
+const Header = () => {
+	return (
+		<div className="flex-hz-space-bw p-2 bg-bg-black md:px-40 sm:px-10 px-5 py-3 border-b border-white border-opacity-50">
+			<p>
+				<img src={logo} alt="SVG" />
```

Comment:
suggestion: in alt you can give logo instead of svg

Reply:
changed to logo